CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 Change log
 -----------
 
+* Update the getting started links [Zach]
 * Update poky to latest krogoth branch [Will]
 * Update meta-rust to morty [Will]
```

Comment:
@zwalchuk I have amended the PR to also include a CHANGELOG.md.
In addition, I also added the "Signed-off-by" entry in the commit log. Please fix your your git client config so that it add the Signed-off-by field automatically for the commits you do